numpy/fft/_pocketfft.py

```diff
@@ -51,10 +51,6 @@ def _raw_fft(a, n, axis, is_real, is_forward, inv_norm):
     if n is None:
         n = a.shape[axis]
 
-    if n < 1:
-        raise ValueError("Invalid number of FFT data points (%d) specified."
-                         % n)
-
```

Comment:
the `ValueError` raise has been moved to the `_go_forw` and `_go_back` functions, since these are the ones to encounter `n` first; shouldn't make a difference, it just helps raise the error at the earliest encounter